src/item_group.cpp

```diff
@@ -148,7 +148,7 @@ Item_spawn_data::ItemList Single_item_creator::create( const time_point &birthda
     if( container_item ) {
         item ctr( *container_item, birthday );
         for( const item &it : result ) {
-            ctr.put_in( it, item_pocket::pocket_type::CONTAINER );
+            ctr.put_in( it );
```

Comment:
rather than this maybe we should have the itemgroup allow you to specify the pocket type.

Reply:
It's probably better. This will allow to spawn items with different pocket types for the same item type. But let it be a separate PR. This PR is only fix for make game playable.

Reply:
Removed put_in and added guess_pocket_for